ui/src/message_popup/renderers/hint_card_test.jsx

```diff
@@ -45,31 +37,4 @@ describe('<HintCard />', () => {
     expect(wrapper.find(RaisedButton).props().label).to.equal("Show another");
     expect(wrapper.contains('exampleText')).to.equal(true);
   });
-
-  it('DOES NOT work to mock methods, because of autobinding', () => {
-    sinon.spy(HintCard.prototype, 'onHintsToggled');
-    const wrapper = shallow(<HintCard examples={['exampleText']} nonExamples={[]} />);
-    wrapper.find(RaisedButton).simulate('touchTap');
-    expect(HintCard.prototype.onHintsToggled.callCount).to.equal(0);
-  });
-
-  it('DOES NOT render VelocityTransitionGroup correctly with full rendering', (done) => {
-    const wrapper = mount(<MuiThemeProvider>
-      <HintCard examples={['exampleText']} nonExamples={[]} />
-    </MuiThemeProvider>);
-    wrapper.find(RaisedButton).simulate('click');
-
-    expect(wrapper.find('.HintCard').length).to.equal(1);
-    setTimeout(() => {
-      expect(wrapper.find(VelocityTransitionGroup).find(RaisedButton).length).to.equal(0);
-      done();
-    }, 1500);
-  });
 });
-
-
-
-// nested integration-style tests
-// rendering inside VelocityTransitionGroup
-// web inspector debugging
-// warning only if prop is missing or the wrong type
```

Comment:
These were my notes when initially setting this up, I pulled them out and will track and tackle them separately.